service/templates/display_title.html

```diff
@@ -142,6 +142,14 @@ <h5 class="{% if loop.index != 1 %} expand-top-half {% endif %}">
 
       </div>
 
+      <div class="printhidden">
+        <h2><strong>What did you think of this service?</strong></h2>
```

Comment:
`<h2 class="heading-small">What did you think of this service?</h2>` would be better here rather than nesting a `<strong>` inside